kubectl-cloudflow/verify/verify.go

```diff
@@ -0,0 +1,142 @@
+package verify
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/go-akka/configuration"
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/docker"
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/domain"
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/util"
+)
+
+// VerifyBlueprint TBD
```

Comment:
Add docs

Reply:
Pls read the related comment in the description it is in the todo list.